src/main/webapp/styles/main.css

```diff
@@ -313,10 +313,10 @@ details[open] summary ~ * {
 
 input[type=date]
 {
-    width:40%;
+    width:42.39%;
```

Comment:
This is very specific haha!

Reply:
Haha please don't change, it will break 😹